spec/player_spec.rb

```diff
@@ -9,7 +9,7 @@
     expect(expected_token).to eq('X')
   end
 
-  it 'can input to interact with the board' do
+  it 'can input to interact with the board and is address player by token' do
     @player = Player.new('X')
```

Comment:
Not sure what this test is getting at...and maybe the description needs to be fixed?

I think you're testing `player.user_input`...which is typically used to get a number from the player, representing a position selection.  (Maybe the method should be renamed to `select_box` or `select_position`?  Or if the method is still to get any input, maybe it should be `make_selection`, since you are "asking" the player object to make a selection of something?)  Anyway, maybe what this boils down to is -- would a user ever provide an array of tokens in response to this method?  If not, maybe make the test clearer -- have it just be an example of selecting a number as a position, with a better test description.  Something like that.  Holler at me if you have any questions.

Reply:
this method was changed to communicate its intentions more clearly